ButterflyMX Demo/Logic/CallsService.swift

```diff
@@ -185,5 +233,57 @@ extension CallsService: PKPushRegistryDelegate, CXProviderDelegate {
     func provider(_ provider: CXProvider, didDeactivate audioSession: AVAudioSession) {
         BMXCallKit.shared.disconnectSoundDevice()
     }
+    
+    func rejectCallForPushNotification(userInfo: [AnyHashable: Any]) {
```

Comment:
when do you call it?

Reply:
Currently when app is in the foreground, the video call will begin automatically if an initial call push notification comes. I wanted to add some simple UI to give users a selection to accept or reject the call, so I added this function. Later I thought it was just a demo app, maybe I don't need to provide this selection. So I didn't add the selection but forgot deleting the function.
What do yo think about it?
Do you think we should provide a selection for users in this scenario?

Reply:
I think we can have a reject button, but would you need to call `BMXCallKit.shared.processCall` to reject it?

Reply:
Yes, I need to get a correct incomingCallProcessor because `BMXCallKit.shared.endCall` calls `processor?.endCall(guid: callId)`

Reply:
hm, that is not obvious and I think can confuse a SDK user, maybe process call should be inside BMXCallKit.shared.endCall then?

Reply:
Good suggestion. I will make changes inside BMXCallKit.shared.endCall then.

Reply:
@sviatoslavbmx I ended up adding a preview call UI for push notification with answer button and reject button, but I don't need rejectCallForPushNotification any more because I will directly use CallsService.shared.endCurrentCall()